tests/helpers/perf_run_helper.rb

```diff
@@ -109,12 +139,13 @@ def create_perf_archive_root
   #   capture_current_tune_settings
   #
   def capture_current_tune_settings
-    puts "Checking current tune settings..."
-    settings_json = run_script_on(master, "./util/tune/current_settings.rb").output
-    output_path = "#{@archive_root}/#{CURRENT_TUNE_SETTINGS}"
-
-    puts "Writing file: #{output_path}"
-    File.write(output_path, settings_json)
+    puts "Generating current tune settings files for each host..."
+    hosts.each do |host|
+      settings_json = run_script_on(host, "./util/tune/current_settings.rb").output
+      role = get_most_relevant_role(host)
+      output_path = "#{@archive_root}/hosts/#{role}/#{host.hostname}/#{CURRENT_TUNE_SETTINGS}"
```

Comment:
I know I added `CURRENT_TUNE_SETTINGS` but maybe it should be updated to `CURRENT_TUNE_SETTINGS_FILENAME` to clarify?

Reply:
updated